File: Python/8kyu/Python/find_needle.py
def find_needle(haystack):
    # your code here
    for item in haystack:
        if item == "needle":
            item_index = haystack.index(item)
            return "found the needle at position " + str(item_index)
print(find_needle(["hay", "junk", "hay", "hay", "moreJunk", "needle", "randomJunk"]))

#You have passed all of the tests! :)      

File: Java/CodewarsJava/src/kyu8/FindNeedle.java
package kyu8;

public class FindNeedle {
    public static String findNeedle(Object[] haystack) {
        for (int i = 0; i < haystack.length; i++) {
            if ("needle".equals(haystack[i])) {
                return "found the needle at position " + i;
            }
    }
        return "";
    }
}
//https://www.codewars.com/kata/56676e8fabd2d1ff3000000c/train/java
//You have passed all of the tests! :)